packages/upload-api/src/blob/accept.js

```diff
@@ -60,3 +62,76 @@ export function blobAcceptProvider(context) {
     },
   })
 }
+
+/**
+ * Polls `blob/accept` whenever we receive a receipt. It may error if passed
```

Comment:
Do you mean "this is called whenever we receive a receipt"? It doesn't do the polling, it is the thing that is polled.

Reply:
```suggestion
 * Polls `blob/accept` task whenever we receive a receipt. It may error if passed
```

Does this make more sense ? For more context I have been modeling invocations after Rust's Async Tasks a.k.a Futures. Scheduler [`poll`](https://doc.rust-lang.org/std/future/trait.Future.html#the-poll-method)s pending tasks (which is modeled after [epoll](https://en.wikipedia.org/wiki/Epoll)) to attempt make progress.

What we do here is very similar, on a system event in form of receipt we poll tasks and progress ones that were awaiting on it.